pylib/anki/collection.py

```diff
@@ -467,6 +481,72 @@ def findDupes(self, fieldName: str, search: str = "") -> List[Tuple[Any, list]]:
     findNotes = find_notes
     findReplace = find_and_replace
 
+    # Search Strings
+    ##########################################################################
+
+    # Helper function for the backend's search string operations.
+    # Pass search strings as 'searches' to normalize.
+    # Pass multiple to concatenate (defaults to 'and').
+    # Pass 'negate=True' to negate the end result.
+    # May raise InvalidInput.
+    def search_string(
+        self,
+        *,
+        negate: bool = False,
+        concat_by_or: bool = False,
+        searches: Optional[List[str]] = None,
+        name: Optional["FilterToSearchIn.NamedFilterValue"] = None,
+        tag: Optional[str] = None,
+        deck: Optional[str] = None,
+        note: Optional[str] = None,
+        template: Optional[int] = None,
+        dupe: Optional[Tuple[int, str]] = None,
+        forgot_in: Optional[int] = None,
+        added_in: Optional[int] = None,
+        due_in: Optional[int] = None,
+        nids: Optional[List[int]] = None,
```

Comment:
These _in feel a bit "half-way" - how about either using just "forgot", "added" etc (matches the other short terms), or the full "forgot_in_days", "added_in_days", etc?

Reply:
I'll go with the full names then to make it clear how the arguments are interpreted.